assets/src/blocks/Splittwocolumns/SplittwocolumnsInPlaceEdit.js

```diff
@@ -0,0 +1,111 @@
+import { Fragment } from '@wordpress/element';
+import { RichText } from '@wordpress/block-editor';
+import { debounce } from 'lodash';
+
+const { __ } = wp.i18n;
+
+/**
+ * WYSIWYG in-place editor 
+ */
+export const SplittwocolumnsInPlaceEdit = ({attributes, charLimit, setAttributes}) => {
+  const toAttribute = attributeName => value => setAttributes({
+    [attributeName]: value
+  });
+  const debounceToAttribute = attributeName => debounce(
+    toAttribute(attributeName), 400
+  );
+
+  return (
+    <Fragment>
+    <section className="block-wide split-two-column">
+      <div className="split-two-column-item item--left">
+        {attributes.issue_image_src &&
+          <div className="split-two-column-item-image">
+            <img
+              src={attributes.issue_image_src}
+              alt={attributes.issue_image_title || ''}
+              style={{objectPosition: attributes.focus_issue_image}}
+            />
+          </div>
+        }
+        <div className="split-two-column-item-content">
+          <RichText
+            tagName="h2"
+            className="split-two-column-item-title"
+            placeholder={__('Enter Title', 'planet4-blocks-backend')}
+            value={attributes.title}
+            onChange={debounceToAttribute('title')}
+            characterLimit={charLimit.title}
+            multiline={false}
```

Comment:
You need to use a string here instead of a boolean for this to work (weird, I know), so: `multiline="false"`... Same for other occurrences in the file 🙂